Web/Edubase.Web.UI/Areas/Governors/Views/Governor/_GovernorEditAppointingBody.cshtml

```diff
@@ -0,0 +1,13 @@
+@using Edubase.Common
+@using Edubase.Services.Enums
+@model Edubase.Web.UI.Areas.Governors.Models.GovernorViewModel
+
+@if (Model.DisplayPolicy.AppointingBodyId)
+{
+    <div class="govuk-form-group @Html.ValidationCssClassFor(x => x.AppointingBodyId)">
+        @Html.LabelFor(x => x.AppointingBodyId, "Appointing body (required to save record)", new { @class = "govuk-label" })
+        @Html.ValidationMessageFor(x => x.AppointingBodyId, null, new { @class = "govuk-error-message" })
+        @Html.DropDownListFor(x => x.AppointingBodyId, Model.AppointingBodies, "",
+            new { id = "governorAppointingBodyInput", @class = string.Concat("govuk-select ", Html.TextBoxValidationClass(x => x.AppointingBodyId)) })
```

Comment:
For info, there's a new Html helper  (@Html.DropdownListWiithIds) that allows custom Ids for each option in the dropdown list. Possibly something that the Test team can replace at some point.

Reply:
Is it as simple as changing @Html.DropDownListFor to @Html.DropdownListWiithIds?

Reply:
It's probably easier to leave this for the test team as the updated helper is new and may not work in all situations if just swapped in.